eos-tech-support/eos-fix-mbr-swap-removal

```diff
@@ -1,4 +1,21 @@
 #!/bin/bash -e
+# Copyright (C) 2018 Endless Mobile, Inc.
+# Licensed under the GPLv2.
+#
+# eos-fix-mbr-swap-removal: Fix a MBR partition table that was corrupted during
+# swap reclaiming.
+#
+# Since we disabled the use of a swap partition on Endless systems, the
+# eos-reclaim-swap service removes the unused swap partition, if existent, and
+# extents the root partition and filesystem to make use of that space. But on
+# systems which were originally installed from an ISO image prior to Endless
+# 3.4.0 that was booted in Legacy (non-UEFI) mode, eos-reclaim-swap ends up
```

Comment:
The triggering conditions listed here are incomplete. I'm going to add some more info to T24493.

Perhaps we can just be unspecific here, I think the rest of the comment is giving enough context about the script. "On some old Endless systems installed under certain conditions, eos-reclaim swap ends up corrupting..." or something

Reply:
I agree with being less specific.